devtale/constants.py

```diff
@@ -13,3 +13,5 @@
 }
 
 DOCSTRING_LABEL = "@DEVTALE-GENERATED:"
+
+GPT_PRICE = {"gpt-4": 0.03, "gpt-3.5-turbo-16k": 0.03, "text-davinci-003": 0.0015}
```

Comment:
prices will change. can you add an inline comment with the date when you got these prices, and a link to the pricing page?